src/v2/Apps/Artists/Components/ArtistsLetterNav.tsx

```diff
@@ -6,13 +6,17 @@ import { RouterLink, RouterLinkProps } from "v2/Artsy/Router/RouterLink"
 
 export const LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("")
 
-interface ArtistsLetterNavProps extends BoxProps {}
+interface ArtistsLetterNavProps extends BoxProps {
+  inDropDown?: boolean
+}
 
 export const ArtistsLetterNav: React.FC<ArtistsLetterNavProps> = ({
+  inDropDown = false,
   ...rest
 }) => {
+  const justifyContent = inDropDown ? "flex-start" : ["flex-start", "flex-end"]
   return (
-    <Flex flexWrap="wrap" justifyContent={["flex-start", "flex-end"]}>
+    <Flex flexWrap="wrap" justifyContent={justifyContent}>
```

Comment:
Seconding @dzucconi, 
```tsx
<Flex flexWrap="wrap" {...rest}>...
```
should do it here, then we don't need the extra prop and can set the `justifyContent` setting higher up:  
```tsx
<ArtistsLetterNav justifyContent="..." />
```